example/presto_standalone/README.md

```diff
@@ -0,0 +1,31 @@
+# Presto cluster example
+The current directory contains terraform related files that use the module in `../../`. The example module spins up presto in [standalone mode](../conf/nomad/presto_standalone.hcl
+) having one node work as both coordinator and worker.
+It uses vault as the shared secret provider. For more details check [main.tf](./main.tf).
+
+## Modules in use
+| Modules       | version       |
+| ------------- |:-------------:|
+| [terraform-nomad-postgres](https://github.com/fredrikhgrelland/terraform-nomad-postgres) | 0.2.0 |
+| [terraform-nomad-minio](https://github.com/fredrikhgrelland/terraform-nomad-minio) | 0.2.0 |
+| [terraform-nomad-hive](https://github.com/fredrikhgrelland/terraform-nomad-hive) | 0.2.0 |
+
+## Services
+![img](./terraform-nomad-presto.png)
```

Comment:
Suggest we move this image to a folder named images/ in the examples folder. That way we don't need to copy it into the other example folders.

Reply:
Maybe under `examples/resources/images/`

Reply:
Yeah, agreed, I'll do it now